lbry/extras/daemon/json_response_encoder.py

```diff
@@ -301,7 +310,7 @@ def encode_file(self, managed_stream):
             'height': tx_height,
             'confirmations': (best_height + 1) - tx_height if tx_height > 0 else tx_height,
             'timestamp': self.ledger.headers[tx_height]['timestamp'] if 0 < tx_height <= best_height else None,
-            'is_fully_reflected': managed_stream.is_fully_reflected
+            'is_fully_reflected': managed_stream.is_fully_reflected if is_stream else False
         }
```

Comment:
Instead of all of those inline ifs can you split this out into:
```
result = { all of the stuff that's the same }
if is_stream:
  result.update({
    all of the stream stuff
  })
if output_exists:
  result.update({
    all of the output exists stuff
  })
...
return result
```